src/containers/TrailerModal/TrailerModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
It looks like there's a lot of overlap between `Cinema` and this component.  I played around with it a bit to see if I could get them into 1 and I think it's possible. Want me to submit a PR to your PR?  Or are they separate for a reason?

Reply:
I kept those separate to make the differences clear. IMHO it gets more complicated when combining functionally different components like this (we had some issues before).

The biggest problem is when changing something for the trailer player, you accidentally change it for the movie as well. 

However, I never mind looking at PRs with suggestions. Please do submit it if you have it ready.

Reply:
OK, I'll submit the PR separate from this.